database/migrations/default/1736197283308_jurisdiction_refactor/up.sql

```diff
@@ -0,0 +1,143 @@
+alter table geo.atd_jurisdictions add column jurisdictions_id integer;
```

Comment:
Is there any way at all we can drop the `atd_` out of this table name? `coa_` is appropriate here, or my true preference: just `jurisdictions`. 

`atd` is no longer, and while `tpw` is marginally better, these jurisdictions belong to the larger city. This similar thing happened with the new VZ S3 bucket name, and it'd be great to avoid this usage here because they are harder to change the longer they remain in place.

Reply:
thank you for nudging me on this! i was a bit worried about blowing up the scope of this, but you're totally right: might as well get this cleaned up once and for all now. i checked with Milad and Xavier and they are ok with this change. i'll push it up!